keps/sig-node/1029-ephemeral-storage-quotas/kep.yaml

```diff
@@ -6,18 +6,25 @@ authors:
 owning-sig: sig-node
 participating-sigs:
   - sig-node
+status: implementable
+creation-date: 2018-09-06
 reviewers:
   - "@dashpole"
   - "@derekwaynecarr"
 approvers:
   - "@dchen1107"
   - "@derekwaynecarr"
-editor: TBD
-creation-date: 2018-09-06
-last-updated: 2023-08-31
-status: implementable
-latest-milestone: "1.29"
+
 stage: "beta"
+
+latest-milestone: "1.31"
+
 milestone:
   alpha: "1.15"
-  beta: "1.29"
+  beta: "1.31"
+
+feature-gates:
+  - name: LocalStorageCapacityIsolationFSQuotaMonitoring
```

Comment:
@soltysh : @PannagaRao and I chatted about this and it feels more right just to include this one as it's net new for this feature. above in the enablement section we do mention UserNamespacesSupport as it's required, but I think it's more correct to just have this feature gate here. WDYT?

Reply:
Fair, let's keep it that way.